test/test.config.js

```diff
@@ -197,7 +197,7 @@ describe('Config API', function() {
                     });
                 });
             } else if (localforage.driver() === localforage.LOCALSTORAGE) {
-                var dbValue = JSON.parse(localStorage['My Cool App/some key']);
+                var dbValue = JSON.parse(localStorage['My Cool App/storeFront/some key']);
```

Comment:
We can certainly change it. I wrote that so long ago I can't remember why, and it's worth changing ;-)

I'll do that outside this PR (or when we merge it in): #453 is filed for it :smile: